### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by ohan-Bioinfo** [Dfam-consortium/RepeatModeler#94]
is the new version of repeatmodele able to identify TIR, DNA Transposons?

**Comment by BioFalcon** [comment]
As they are included in Dfam I would assume they do identify them :)

**Comment by rmhubley** [comment]
Yes, RepeatModeler should be able to identify non-autonomous DNA transposon deletion products ( the class "TIR" was simply assigned at first when it was unclear where these sequences originated from -- unfortunately the name stuck ).  What matters for RepeatModeler is sequence homology and copy number and not structure.
